src/FantasyLand/Monoid.php

```diff
@@ -4,6 +4,11 @@
 interface Monoid extends Semigroup
 {
     /**
+     * @return Monoid
+     */
+    public static function mempty();
```

Comment:
Why add a method here which should return the exact same value as `getEmpty` if I understand correctly ?